cogs/reddit.py

```diff
@@ -192,6 +195,37 @@ async def remove_entry(
             ephemeral=True,
         )
 
+    async def create_modmail_embed(
+        self, modmail: asyncpraw.models.ModmailConversation
+    ) -> discord.Embed:
+        embed = discord.Embed(
+            title=f"new modmail: {modmail.subject[:100]}",
+            description=modmail.messages[-1][:4000],
+        )
+        embed.set_author(
+            name=modmail.participant.name, icon_url=modmail.participant.icon_img
+        )
+        return embed
+
+    @tasks.loop(minutes=1)
+    async def get_modmail(self):
+        new_conversations: typing.AsyncIterator[
+            asyncpraw.models.ModmailConversation
+        ] = self.subreddit.modmail.conversations(sort="unread", state="all")
+
+        async for conv in new_conversations:
+            embed = self.create_modmail_embed(conv)
+            await self.bot.modqueue_channel.send(embed=embed)
```

Comment:
Please don't use the modqueue channel for modmail. Make a separate env var for the modmail channel